### PR TITLE
Feature/we 588 implement dotenv for configuration

### PR DESCRIPTION
**Pull request by duncannixon**
The purpose of this PR is to use the Dotenv framework to read in our application config from a local .env file.

See: https://www.npmjs.com/package/dotenv

The following changes were made:

- Added the Dotenv framework for managing our application configuration. 
- Created and git-ignored a local .env config file.
- Added a .env.example file.
- Confirmed that the config is working by changing the app to use a port number from the config rather than a hard-coded port number.

See https://eaflood.atlassian.net/browse/WE-588

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# You'll need to set the following in either your environment or in a .env
+# file in the root of the project
+#
+# The values in this file are purely examples to better help understand what it
+# is you need to add
+
+PORT=8000
```
Comment (Cruikshanks) [inline .env.example]: I know its obvious now, but for the possible future me that has to get this thing running with no one to hold my hand 😟 , could we add a brief comment as to what port we are referring to.

Also should we be more explicit in the name e.g. WP_PORT or WASTE_PERMITS_PORT or THE_WEB_APPLICATION_SERVER_PORT_FOR_THE_WASTE_PERMITS_DIGITAL_SERVICE. Open to suggestions (though don't think you'll beat my last one!)

File: index.js [hunk 2]
```diff
@@ -6,10 +6,13 @@ const HapiAlive = require('hapi-alive')
 const HapiDevErrors = require('hapi-dev-errors')
 const server = new Hapi.Server()
 
+// Load application configuration using Dotenv (see https://www.npmjs.com/package/dotenv)
+require('dotenv').config()
+
 const environment = process.env.NODE_ENV
```
Comment (Cruikshanks) [inline index.js]: Not a change to this line, but can be include an entry for this in the `.env.example` file.

File: .env.example (hunk 1 above)
Comment (duncannixon) [inline .env.example]: No problem, I have added a comment and changed the name to: WASTE_PERMITS_APP_PORT

File: index.js (hunk 2 above)
Comment (duncannixon) [inline index.js]: Sure -I have added the comment to the .env.example file

**Comment by Cruikshanks** [comment]
:shipit:

**Pull request merged**